Mlem/App/Views/Shared/Palette Components/Form.swift

```diff
@@ -22,7 +22,9 @@ struct Form<Content: View>: View {
                 .tint(palette.accent)
                 .buttonStyle(PaletteButton())
         }
+        .listStyle(InsetListStyle())
         .scrollContentBackground(.hidden)
         .background(palette.groupedBackground)
+        .shadow(color: palette.primary.opacity(palette.bordered ? 0.2 : 0.0), radius: 5)
```

Comment:
This shadow shows up on `Form` sections for which we removed the background using `.listRowBackground`. Possibly fixable by applying another `.shadow` on those views or something?

Reply:
Nice catch--this one was a little hacky, I had to move the content into the header of an empty Section. This does seem to fix the spacing issues that we were previously solving with negative padding.